Doc/library/json.rst

```diff
@@ -433,15 +433,15 @@ Encoders and Decoders
    .. versionchanged:: 3.6
       All parameters are now :ref:`keyword-only <keyword-only_parameter>`.
 
-   .. method:: decode(s)
+   .. method:: decode(s, _w=WHITESPACE.match)
```

Comment:
This is half an implementation detail half a feature. The reason is that callers or subclasses could change how whitespaces are matched by default. In addition `WHITESPACE` is not documented at all so this would be more confusing than helping.

I suggest we do not change this in this PR and decide whether to expose `_w` publicly or whether to indicate it explicitly as being `internal` only.

Reply:
ok, so i will also remove the `_one_shot` and `idx` arguments